database/mysql/mysql.go

```diff
@@ -207,6 +208,7 @@ func (m *Mysql) Open(url string) (database.Driver, error) {
 	mx, err := WithInstance(db, &Config{
 		DatabaseName:    config.DBName,
 		MigrationsTable: customParams["x-migrations-table"],
+		NoLock:          customParams["x-no-lock"] == "true",
```

Comment:
Use [strconv.ParseBool()](https://golang.org/pkg/strconv/#ParseBool) and properly handle errors. e.g. fail on errors

Reply:
Yeah that is much better! Fixed. Added a test too, maybe a bit excessive - your call :)